Space-Acres.app/Contents/Info.plist

```diff
@@ -5,7 +5,7 @@
     <key>CFBundleExecutable</key>
     <string>space-acres</string>
     <key>CFBundleIdentifier</key>
-    <string>com.subspace.spaceacres</string>
+    <string>org.subspace.spaceacres</string>
```

Comment:
Also I'd probably prefer `space-acres` if it is supported for consistency with everything else

Reply:
Fixed in my branch, thank you.